ingest/validation/validate_metadata.py

```diff
@@ -120,7 +122,38 @@ def merge_numerics(metadata):
     return
 
 
+def list_duplicates(cells):
+    """Find duplicates in list for detailed reporting
+    """
+    # reference https://stackoverflow.com/questions/9835762
+    seen = set()
+    # save add function to avoid repeated lookups
+    seen_add = seen.add
+    # adds all new elements to seen and all other to seen_twice
+    seen_twice = set(x for x in cells if x in seen or seen_add(x))
```

Comment:
I'd love to get the explanation for what's actually happening here...

Reply:
added comments in d86fc7c
    # list comprehension below chosen for computational efficiency
    # adds all new elements to seen and all other to seen_twice
    # "or" allows seen_add(x) evaluation only if x is not in seen